--- a/src/newsrc.c
+++ b/src/newsrc.c
@@ -756,7 +756,7 @@
   }
 
   while (fgets (b,MAX_BUF,new)) 
-    fprintf (old, b);
+    fprintf (old,"%s", b);
 
   fclose (old);
   fclose (new);
@@ -854,7 +854,7 @@
     seq[0] = 0;
     newsrc_writeseq (n,seq,MAX_BUF);
     sprintf (b, "%s%c %s\n", n->name, (n->attribs&naSUBSCRIBED)?':':'!', seq);
-    fprintf (new, b);
+    fprintf (new,"%s", b);
     n->attribs &= ~(naUPDATE);
     n = nextgroup (n);
   }

